GUI-dev/src/models/models.ts

```diff
@@ -5,6 +5,19 @@ export class Process {
   // define props...
 }
 
+export class ProcessStore {
+  processId: number;
+  processName: string;
+  processDescription: string;
+  processCreator: string;
+  processCreatedAt: Date;
+  processVersion: number;
+  processPrice: number;
+  // missing in current ProcessStoreDTO in backend:
```

Comment:
We are not 100% sure if we will ever show any image of the actual process.
However, it was initially a nice idea to implement something like this.
I would rather suggest to remove it when doing the final adaptations.

`processAverageRating` is a good idea, but it is also not clear how this will be calculated in the backend.
From my point of view however, it will most likely be provided via a separate endpoint which gives you the average rating of all ratings.